client/src/Movies/MovieList.js

```diff
@@ -1,5 +1,7 @@
 import React, { Component } from 'react';
 import axios from 'axios';
+import { BrowserRouter as Router, Route, Link } from "react-router-dom";
```

Comment:
Same as App.js, no need for BrowserRouter here or Route since you're not nesting any routes in this file.